src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -306,11 +303,25 @@ public void createGoogleContextSpendWithoutBillingAccountThrows() {
   }
 
   /**
-   * Generate a Workspace object with a random ID and default values in all fields.
+   * Convenience method for getting a WorkspaceRequest builder with some pre-filled default values.
+   *
+   * <p>This provides default values for operationID (random UUID), spend profile
+   * (Optional.empty()), and workspace stage (RAWLS_WORKSPACE).
+   */
+  private WorkspaceRequest.Builder defaultRequestBuilder(UUID workspaceId) {
+    return WorkspaceRequest.builder()
+        .workspaceId(workspaceId)
+        .jobId(UUID.randomUUID().toString())
+        .spendProfileId(Optional.empty())
+        .workspaceStage(WorkspaceStage.RAWLS_WORKSPACE);
+  }
+
+  /**
+   * Generate a fully built Workspace object with a random ID and default values in all fields.
    *
-   * <p>See {@code WorkspaceRequest.defaultRequestBuilder} for default values of each field.
+   * <p>See {@code defaultRequestBuilder} for default values of each field.
    */
   private WorkspaceRequest defaultWorkspaceRequest() {
-    return WorkspaceRequest.defaultRequestBuilder(UUID.randomUUID()).build();
+    return defaultRequestBuilder(UUID.randomUUID()).build();
   }
```

Comment:
multiple factory functions when there is a builder object is an anti-pattern that's caused me personal grief before. Let's pick one way to make a `default` `WorkspaceRequest.Builder` and customize that as appropriate. i.e. remove the (UUID workspaceId) argument method and call `defaultWorkspaceRequest().workspaceId(foo)` if needed.

Reply:
I think that points towards leaving the builder and removing the `defaultWorkspaceRequest` since AutoValue objects are immutable, but sticking to a single factory function makes sense.